vector-test/src/test/scala/geotrellis/vector/spec/json/GeoJsonSpec.scala

```diff
@@ -68,6 +68,7 @@ class GeoJsonSpec extends FlatSpec with Matchers {
 
     val json="""{"type":"FeatureCollection","features":[{"type":"Feature","geometry":{"type":"Point","coordinates":[2674010.3642432094,264342.94293908775]},"properties":{"data":291}},{"type":"Feature","geometry":{"type":"Point","coordinates":[2714118.684319839,263231.3878492862]},"properties":{"data":1273}}]}"""
 
+    println(json.parseGeoJson[JsonFeatureCollection].getAllPointFeatures[DataBox])
```

Comment:
if you use info() instead of println, ScalaTest is going to include that readout along with the test list rather than the spam scroll